k8s/demo/common/bsp/bulk-scan-sample-app.yaml

```diff
@@ -20,7 +20,7 @@ spec:
     java:
       replicas: 2
       useInterpodAntiAffinity: true
-      image: hmctspublic.azurecr.io/bulk-scan/sample-app:prod-84f169c-20210915082635
+      image: hmctspublic.azurecr.io/bulk-scan/sample-app:prod-84f169c-20210915082635   #{"$imagepolicy": "flux-system:bulk-scan-sample-app"}
```

Comment:
Should this policy not be `bsp-sample-app`

Reply:
bulk-scan-sample-app now removed as product chart no longer required.